tests/buyers/views/test_buyers.py

```diff
@@ -1932,16 +1934,19 @@ def test_api_error(self, data_api_client):
 
 
 class AbstractViewBriefResponsesPage(BaseApplicationTest):
-    def test_page_shows_correct_count_of_eligible_suppliers(self, data_api_client):
-        data_api_client.find_brief_responses.return_value = self.two_good_three_bad_responses
+    def test_page_shows_correct_content_when_eligible_responses(self, data_api_client):
+        data_api_client.find_brief_responses.return_value = self.brief_responses
         data_api_client.get_framework.return_value = api_stubs.framework(
             slug='digital-outcomes-and-specialists',
             status='live',
             lots=[
                 api_stubs.lot(slug='digital-outcomes', allows_brief=True),
             ]
         )
-        data_api_client.get_brief.return_value = api_stubs.brief(lot_slug="digital-outcomes")
+
+        brief_stub = api_stubs.brief(lot_slug="digital-outcomes")
+        brief_stub['briefs'].update({'publishedAt': '2017-01-21T12:00:00.000000Z'})
```

Comment:
This common test is shared between the new and old flow test classes but this date is just applicable for one. 

You could put something like `self.briefPublishedAt` value on both the `TestViewBriefResponsesPageForLegacyBrief` and `TestViewBriefResponsesPageForNewFlowBrief` and then in this test set `brief_stub['briefs'].update({'publishedAt': self.briefPublishedAt})`.

Maybe this date has just been put here because you need published at to be "something" but the date doesn't actually matter for the common tests, however I think that means your tests then rely on your implementation